### PR TITLE
sequela with no dw data return zeroes

### PR DESCRIPTION
**Pull request by halfhorst**
If metadata tells us a sequela has no disability weight associated with it, assign it zeroes.

### CODE REVIEW

File: src/vivarium_inputs/core.py [hunk 1]
```diff
@@ -109,7 +109,12 @@ def get_disability_weight(entity: Union[Cause, Sequela], location_id: int) -> pd
                 data += prevalence * disability
         data = data.reset_index()
     else:  # entity.kind == 'sequela'
-        data = extract.extract_data(entity, 'disability_weight', location_id)
+        if not entity.healthstate.disability_weight_exists:
+            cols = {'location_id': [location_id], 'sex_id': [3], 'age_group_id': [22]}
```
Comment (kfwilson) [inline src/vivarium_inputs/core.py]: can we use constants or at least named vars for 3 and 22?

Comment (halfhorst) [reply src/vivarium_inputs/core.py]: used  named vars unless we decide we want to introduce these as globals

File: src/vivarium_inputs/core.py [hunk 2]
```diff
@@ -109,7 +109,14 @@ def get_disability_weight(entity: Union[Cause, Sequela], location_id: int) -> pd
                 data += prevalence * disability
         data = data.reset_index()
     else:  # entity.kind == 'sequela'
-        data = extract.extract_data(entity, 'disability_weight', location_id)
+        if not entity.healthstate.disability_weight_exists:
+            all_ages_id = 22
```
Comment (collijk) [inline src/vivarium_inputs/core.py]: Put these constants in `globals` (also grep for 22 and 3 and replace them with the constant).

Comment (collijk) [reply src/vivarium_inputs/core.py]: You could also use `utilities.get_demographic_dimensions` to get the full demography, and append a len(demography) x len(draws) block of zeroes to the returned frame.

File: src/vivarium_inputs/core.py [hunk 3]
```diff
@@ -108,7 +108,12 @@ def get_disability_weight(entity: Union[Cause, Sequela], location_id: int) -> pd
                 data += prevalence * disability
         data = data.reset_index()
     else:  # entity.kind == 'sequela'
-        data = extract.extract_data(entity, 'disability_weight', location_id)
+        if not entity.healthstate.disability_weight_exists:
+            full_demog = utilities.get_demographic_dimensions(location_id)
+            draws = pd.DataFrame({f'draw_{i}': len(full_demog) * [0.0] for i in range(1000)})
```
Comment (collijk) [inline src/vivarium_inputs/core.py]: Use the ``DRAW_COLUMNS`` global for the column names.

**Pull request merged**